.drone.yml

```diff
@@ -333,27 +332,27 @@ steps:
       - kind get kubeconfig --name $${CLUSTER_NAME} > $${KUBECONFIG}
 
   - name: e2e
-    # BATS 1.1.0 - HELM 3.12.0 - KUBECTL 1.30.5 - KUSTOMIZE 3.10.0 - YQ 4.33.3
-    image: quay.io/sighup/e2e-testing:1.1.0_3.12.0_1.30.5_3.10.0_4.33.3
+    # KUBECTL 1.32.5 - KUSTOMIZE 3.5.3 - HELM 3.1.1 - YQ 4.21.1 - ISTIOCTL 1.9.4 - FURYCTL 0.9.0 - BATS 1.1.0
```

Comment:
nitpick: this comment is not 100% right, for example it say kustomize 3.5.3 when the image uses 5.6.0